tests/integration/conftest.py

```diff
@@ -10,36 +10,51 @@
 import re
 import typing
 
-import juju.action
-import juju.application
 import kubernetes
 import pytest
 import pytest_asyncio
-import pytest_operator.plugin
 import swiftclient
 import swiftclient.exceptions
 import swiftclient.service
+from juju.action import Action
+from juju.application import Application
+from juju.model import Model
+from pytest import FixtureRequest
+from pytest_operator.plugin import OpsTest
 
 from tests.integration.wordpress_client_for_test import WordpressClient
 
 logger = logging.getLogger()
 
 
+@pytest.fixture(scope="module", name="model")
+def model_module_scope_fixture(ops_test: OpsTest) -> Model:
+    """Get current valid model created for integraion testing with module scope."""
+    assert ops_test.model
+    return ops_test.model
+
+
+@pytest.fixture(scope="function", name="func_model")
```

Comment:
Are both of these needed? Shouldn't one of these be enough? I'm guessing the module scope fixture can also be used where a function scope fixture would otherwise be used?

Reply:
Pytest complains about the scope being incompatible when using module scoped fixture in function scoped test/fixture, would there be another way around it?